integ-test/src/test/java/org/opensearch/sql/ppl/DataTypeIT.java

```diff
@@ -48,8 +48,8 @@ public void test_nonnumeric_data_types() throws IOException {
         schema("keyword_value", "string"),
         schema("text_value", "string"),
         schema("binary_value", "binary"),
-        schema("date_value", "timestamp"),
-        schema("date_nanos_value", "timestamp"),
+        schema("date_value", "date"),
```

Comment:
why change timestmap to date? IT is broken?

Reply:
Yes now that date fields are type `OpenSearchDateType` and not ` ExprCoreType`, I was having IT tests mismatch, this is to fix that.